### DISCUSSION TITLE
How to enable TLS1.0 on latest alpine3.15

### DISCUSSION THREAD

**Discussion by mrhovunping** [dotnet/dotnet-docker#3588] [Q&A]
Hi, 

We plan to move to alpine3.15 but we have an issue with our current infra. Some of our internal db are still on sqlserver 2008 and this kind of server only accept tls1.0. With alpine3.14, we are able to enable support for tls1.0 updating the /etc/ssl/openssl.cnf file.

On alpine3.15 this file has been removed. 

Can you help us to solve our problem?

Regards,

**Comment by omajid** [comment]
I think you want to use the `/etc/ssl1.1/openssl.cnf` file?

`/etc/ssl/openssl.cnf` still exists, but it's for configuring OpenSSL 3.0:

```
/ # apk info --who-owns /etc/ssl/openssl.cnf
/etc/ssl/openssl.cnf is owned by libcrypto3-3.0.2-r0
```

**Reply by mrhovunping** [reply]
Hi, 

Thanks for you quick answer, yes, this is the one. 

I have setup the value : 

[system_default_sect]
MinProtocol = TLSv1
CipherString = DEFAULT@SECLEVEL=1

Similar to what we did on alpine3.14 but it doesn't work our same app code can't connect to our 2008 server. 
Idea why?

regards,
